### DISCUSSION TITLE
Subcategories for discussions

### DISCUSSION THREAD

**Discussion by tommygiesbrecht** [community/community#32323] [Discussions] [answered]
Hey community 👋

I really like the discussions feature and want to use it for internal communication. But I can't find the option to create subcategories like in the GitHub Product Feedback category.  

![image](https://user-images.githubusercontent.com/67358982/189159909-1c5866d0-ec85-41e6-88c4-35e1e21a3b91.png)

**Comment by kaosine** [comment]
Please @github-staff implement this site wide. Sub categories would help tremendously....or at least if in an org, and we enable discussions in the org, and also in a repo or team let it surface as a category w/sub-categories in the main org discussions.....it's just too bare bones and most people won't consider using alternatives bc this exists even if it's a pain to use as is...

**Comment by ChrisAcrobat** [comment]
I was also searching for this. I would also like to place the categories in a custom order, but that is an separate issue.

**Comment by LindseyB** [comment]
Presently, this is only available for github.com/community. Definitely will be considered to be released more broadly.

**Reply by amacneil** [reply]
Please release more broadly! The fact that you needed it yourself should be an obvious indicator that others need this feature.

**Reply by mavober** [reply]
+1

**Comment by evilaliv3** [comment]
Hello @LindseyB ,

i'm evaluating the possibilities of internationalization of the GitHub discussions and i notice that a feature of Subcategories like the one already implemented for @community would be very beneficiary to enable Discussions maintainers to organize discussions by language.

This could be an interesting feature to short-come the limits of GitHub discussions in relation to the localization/internationalization expained here: https://github.com/orgs/community/discussions/45769 https://github.com/orgs/community/discussions/45770

Thank you!

**Comment by mavober** [comment]
This feature would be a game changer for the company I work with. We are trying to stop using Slack as the main communication channel for engineering and product. Sub-categories would help us have categories for each internal product team. For example, the CICD team could have a category for roadmap discussions, one for help requests and one for feature requests. I'm a big fan of discussions, feel free to contact me if you want to explore the topic together. In the last 2 months we have created over 200 discussion months.

**Comment by jimmo** [comment, marked as answer]
For anyone following this -- I just noticed this feature is now generally available (or at least is for us). Haven't seen any announcement from GitHub.

You can now place categories into "sections". Works perfectly for us. Thanks GitHub team!

**Reply by ChrisAcrobat** [reply]
Can confirm, thanks for the heads-up!

**Reply by shvchk** [reply]
Yep, but you can't link to section or filter by it yet, here's a relevant FR: #69274

**Reply by jsoref** [reply]
Here's the doc:
https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions#creating-a-section
https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions#creating-a-category

Here's the introduction:
https://github.com/github/docs/commit/d3fde86d86bd5e02f6f76cf24998073c03a868f1

Here's when the docs arrived:
https://github.com/github/docs/pull/25064

Changelog entry: https://github.blog/changelog/2023-04-17-organize-discussions-categories-into-sections/

**Reply by jsoref** [reply]
@tommygiesbrecht please mark https://github.com/orgs/community/discussions/32323#discussioncomment-5868933 as the answer
